XXX

--- src/Common/IpOptionsList.hpp
+++ src/Common/IpOptionsList.hpp
@@ -12,7 +12,7 @@
 #include "IpException.hpp"
 #include "IpRegOptions.hpp"
 
-#include <iostream>
+#include <istream>
 #include <map>
 
 namespace Ipopt
--- src/Interfaces/IpIpoptApplication.hpp
+++ src/Interfaces/IpIpoptApplication.hpp
@@ -7,7 +7,7 @@
 #ifndef __IPIPOPTAPPLICATION_HPP__
 #define __IPIPOPTAPPLICATION_HPP__
 
-#include <iostream>
+#include <istream>
 
 #include "IpJournalist.hpp"
 #include "IpTNLP.hpp"
